conftest.py

```diff
@@ -9,6 +9,8 @@
 settings.register_profile("debug", max_examples=10, verbosity=Verbosity.verbose)
 settings.load_profile(os.getenv("HYPOTHESIS_PROFILE", "default"))
 
+collect_ignore = ["noxfile.py"]
```

Comment:
❔ I wasn't able to get `pytest` to ignore this file from within `pyproject.toml`, since I could only find options for ignoring directories rather than files.  

It should also help to have a global pytest configuration file so that it works for both regular tests (in `tests/`) and doctests (in `src/plasmapy/`).

Reply:
Hmm 🤔 how strange